winml/lib/Api.Ort/OnnxruntimeEngineBuilder.cpp

```diff
@@ -37,8 +37,9 @@ STDMETHODIMP OnnxruntimeEngineBuilder::CreateEngine(Windows::AI::MachineLearning
   auto session_options = UniqueOrtSessionOptions(ort_options, ort_api->ReleaseSessionOptions);
 
   if (batch_size_override_.has_value()) {
-    constexpr const char* DATA_BATCH = "DATA_BATCH";
-    RETURN_HR_IF_NOT_OK_MSG(ort_api->AddFreeDimensionOverride(session_options.get(), DATA_BATCH, batch_size_override_.value()),
+    // This asks the model is using "n" as the batch dim parameter string.
```

Comment:
Should this still align with https://github.com/onnx/onnx/blob/master/docs/DimensionDenotation.md and use DATA_BATCH?

Reply:
no. Per the agreement with Jeff, the API should be using dimension parameter instead of denotation. This is also what winml is asking.